Client/src/Pages/Settings/index.jsx

```diff
@@ -1,9 +1,978 @@
-import React from 'react'
+import PropTypes from "prop-types";
+import { useState } from "react";
+import {
+  Box,
+  Typography,
+  Tab,
+  useTheme,
+  TextField,
+  Avatar,
+  Stack,
+  Divider,
+  Modal,
+  FormControl,
+  OutlinedInput,
+  InputAdornment,
+  IconButton,
+  Container,
+  Table,
+  TableHead,
+  TableRow,
+  TableCell,
+  TableBody,
+  Checkbox,
+  Select,
+  MenuItem,
+} from "@mui/material";
+import TabPanel from "@mui/lab/TabPanel";
+import TabContext from "@mui/lab/TabContext";
+import TabList from "@mui/lab/TabList";
+import Visibility from "@mui/icons-material/Visibility";
+import VisibilityOff from "@mui/icons-material/VisibilityOff";
+import InfoOutlinedIcon from "@mui/icons-material/InfoOutlined";
+import WarningAmberOutlinedIcon from "@mui/icons-material/WarningAmberOutlined";
+import ButtonSpinner from "../../Components/ButtonSpinner";
+import Button from "../../Components/Button";
+import "./index.css";
+import AnnouncementsDualButtonWithIcon from "../../Components/Announcements/AnnouncementsDualButtonWithIcon/AnnouncementsDualButtonWithIcon";
+
+const tabList = ["Profile", "Password", "Team"];
+const profileConfig = {
+  firstName: {
+    id: "edit-first-name",
+    label: "First name",
+    type: "input",
+  },
+  lastName: {
+    id: "edit-last-name",
+    label: "Last name",
+    type: "input",
+  },
+  email: {
+    id: "edit-email",
+    label: "Email",
+    description: "After updating, you'll receive a confirmation email.",
+    type: "input",
+  },
+  photo: {
+    id: "edit-photo",
+    label: "Your photo",
+    description: "This photo will be displayed in your profile page.",
+    type: "photo",
+  },
+};
```

Comment:
I'm not sure we need a config object here, these fields are never going to change, we can probably hardcode them in?

I generally approve of keeping this as modular as possible but I think in this case the downside of the component being harder to read outweighs the benefits of keeping this config separate from the UI

Reply:
Yea you're not wrong. The reason why I added the config was in case we wanted to add more fields in the future and it would be easily accessible. Will remove on my next pr.

Reply:
Makes sense!